### PR TITLE
Chewy swap patch 1

### PR DESCRIPTION
**Pull request by ChewySwap**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### List of audit links if any:  https://github.com/ContractWolf/smart-contract-audits/blob/main/ContractWolf_Audit_ChewySwap_DEX.pdf
https://github.com/ContractWolf/smart-contract-audits/blob/main/ContractWolf_Audit_DogeShrekDEX.pdf

##### Github org/user (Optional, if your code is open source, we can track activity): https://github.com/chewyswap

### CODE REVIEW

File: projects/ChewySwap/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const { uniTvlExport } = require('../helper/unknownTokens')
+
+module.exports = {
+  shibarium: {
+    tvl: getUniTVL({ factory: "0x5640113EA7F369E6DAFbe54cBb1406E5BF153E90", useDefaultCoreAssets: true,})
+  },
+  dogechain: {
+    tvl: getUniTVL({ factory: "0x7C10a3b7EcD42dd7D79C0b9d58dDB812f92B574A", useDefaultCoreAssets: true,})
```
Comment (g1nt0ki) [inline projects/ChewySwap/index.js:8]: this factory belongs to DogeShrek? https://defillama.com/protocol/doge-shrek

File: projects/ChewySwap/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const { uniTvlExport } = require('../helper/unknownTokens')
+
+module.exports = {
+  shibarium: {
+    tvl: getUniTVL({ factory: "0x5640113EA7F369E6DAFbe54cBb1406E5BF153E90", useDefaultCoreAssets: true,})
```
Comment (g1nt0ki) [inline projects/ChewySwap/index.js:5]: this belongs to punkswap? https://defillama.com/protocol/punkswap

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

**Comment by ChewySwap** [comment]
Oof, I'm so sorry it was late when I did this PR. I meant to put 
chewyswap's factory in there and Dogeshrek has actually being merged 
into ChewySwap. I took over operations of the dex back in July and we 
rebranded. So the Dogechain factory is correct the Dogeshrek page can be 
removed and replaced with ChewySwap.

ChewySwap Shibarium: 0xEDedDbde5ffA62545eDF97054edC11013ED72125

ChewySwap Dogechain: 0x7C10a3b7EcD42dd7D79C0b9d58dDB812f92B574A
(Formerly Dogeshrek)

https://docs.chewyswap.com/developers/smart-contracts/chewyswap-exchange

So sorry for the mixup didn't mean to create more work for you. Should I 
update the PR? As proof of the merge if you look at the read contract 
page of each factory contract you will see they have the same feeTo 
address which is the deployer wallet for ChewySwap and new dev wallet 
for Dogeshrek. If there's anything else I need to do to show proof of 
the merge let me know.

Thanks,
Jim

**Comment by g1nt0ki** [comment]
closing as https://github.com/DefiLlama/DefiLlama-Adapters/pull/7829

**Pull request closed without merging**